src/templates/type/moment-select.html

```diff
@@ -7,7 +7,7 @@
                         class="select"
                         ng-model="hours"
                         ng-disabled="disable"
-                        ng-options="hour as hour for hour in hourChoices">
+                        ng-options="hour as (trailingHours + hour).slice(-(trailingHours.length)) for hour in hourChoices">
```

Comment:
I think my example is confusing because of the `trailHours` boolean. You could instead format the `hourChoices` for use in the template.

Ideally it would be:
```html
<select ng-options="hour as (hour | trailingZeros:2) for hour in hourChoices"></select>
```